services/brig/src/Brig/API/Error.hs

```diff
@@ -191,6 +191,9 @@ authError AuthPendingInvitation = StdError accountPending
 reauthError :: ReAuthError -> Error
 reauthError ReAuthMissingPassword = StdError (errorDescriptionTypeToWai @MissingAuth)
 reauthError (ReAuthError e) = authError e
+reauthError ReAuthNoPendingCode = StdError verificationCodeAuthFailed
+reauthError ReAuthVerificationCodeRequired = StdError verificationCodeRequired
+reauthError ReAuthNoEmail = StdError verificationCodeAuthFailed
```

Comment:
Distinguish the first and the last at least in the free-text message?

Reply:
Missing email is a possible failure when verifying a code, but the API spec doesn't have this, and it is very unlikely this ever happens, so that's why, I would leave it like this.

Reply:
ed7043d7b35a419d37b16662657aaa17f1bba089

sorry :)